chalicelib/checks/helpers/cgap_utils.py

```diff
@@ -92,33 +96,30 @@
         "run_time": 12,
         "accepted_versions": ["v13"]
     },
-    "workflow_mutanno-micro-annot-check": {
+    # Part III
+    "workflow_granite-rckTar": {
         "run_time": 12,
         "accepted_versions": ["v13"]
     },
-    "workflow_granite-whiteList-check": {
+    "workflow_mutanno-micro-annot-check": {
         "run_time": 12,
-        "accepted_versions": ["v13"]
+        "accepted_versions": ["v13", "v14"]
```

Comment:
Only v15 should be accepted for micro-annot (many things changed between v13 and v14 and v15)